ledger-service/cli-opts/BUILD.bazel

```diff
@@ -16,7 +16,15 @@ da_scala_library(
     ],
     scalacopts = lf_scalacopts,
     tags = ["maven_coordinates=com.daml:http-json-cli-opts:__VERSION__"],
+    versioned_scala_deps = {
+        "2.12": [
+            "@maven//:org_scala_lang_modules_scala_collection_compat",
+        ],
+    },
     visibility = ["//visibility:public"],
+    runtime_deps = [
+        "@maven//:org_codehaus_janino_janino",
```

Comment:
have you checked that this work via `daml-sdk-head`? We bundle everything into a single JAR there to save size and we might also need to add it to the runtime deps of that jar.

Reply:
Got no problems